saviSimulation/savi/asl/working.asl

```diff
@@ -83,8 +83,11 @@ targetFar :-
 	:	target(threat,TYPE,AZ,EL,RANGE) & 
 		position(X_REF,Y_REF,Z_REF,_) &
 		velocity(BEARING,_,_,_)
-	<-	savi.UxVInternalActions.GetAbsolutePosition(X_TARGET,Y_TARGET,Z_TARGET,X_REF,Y_REF,Z_REF,BEARING,AZ,EL,RANGE);
-		.broadcast(tell,threatSeen(X_TARGET,Y_TARGET,Z_TARGET)).
+	<-	.broadcast(tell,threatSeen(3,4));
```

Comment:
I think the changes to this file should be reverted. Ideally, we will want to have the talkative behaviour and this behaviour merge so that the agent is doing useful things and also saying useful things.